services/src/messaging/src/main/java/org/openkilda/messaging/model/FlowPairDto.java

```diff
@@ -25,9 +25,8 @@
 import java.util.Objects;
 
 @JsonSerialize
-@SuppressWarnings("squid:S1948")
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public class FlowPair<L, R> implements Serializable {
+public class FlowPairDto<L, R> implements Serializable {
```

Comment:
If it is FlowPair it must not be generic... must be able to "keep" only `FlowDto` objects.

Reply:
This is going to be fixed in the future data model.